### PR TITLE
Add banner instructions

### PR DESCRIPTION
**Pull request by frossie**

### CODE REVIEW

**Review by ctslater — APPROVED** [review]
Looks great, this is super useful, thanks.

File: docs/admin/outage-comms.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    ---
+
+    We are aware of and are investigating an issue with DP0.2 searches in the portal
+
+The summary appears on the banner, the body appears if the user clicks the "details" button on the banner.
+See :doc:`environments </environments/index>` for the short labels for your environment.
+
+.. seealso::
+
+    More information on semaphore and the syntax of its payload repositories can be found at https://sqr-060.lsst.io.
```
Comment (jonathansick) [inline docs/admin/outage-comms.rst:57]:
```suggestion
    More information about the syntax of Semaphore broadcasts messages can be found in the `rsp_broadcast README <https://github.com/lsst-sqre/rsp_broadcast/blob/main/README.md>`__.
    Information about Semaphore's installation and operation can be found in `the Semaphore documentation <https://github.com/lsst-sqre/rsp_broadcast/blob/main/README.md>`__.
```

I'd rather not point users to the design technote. Hence the suggested replacement of these links with the intended user and operator docs.

File: docs/admin/outage-comms.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    When changing messages on rsp_broadcast specifically you are allowed to:
+
+    * Commit directly to main
+    * Self-merge / skip review
+    * "Edit on Github"
+
+Steps:
+------
+
+1. Edit or create a message in https://github.com/lsst-sqre/rsp_broadcast/tree/main/broadcasts (if you are in a hurry and you have an issue affecting data.lsst.cloud consider editing 104_idfprod_outage.md on Github and committing directly to main)
```
Comment (jonathansick) [inline docs/admin/outage-comms.rst:26]:
```suggestion
1. Edit or create a message in https://github.com/lsst-sqre/rsp_broadcast/tree/main/broadcasts (if you are in a hurry and you have an issue affecting data.lsst.cloud consider editing 104_idfprod_outage.md on GitHub and committing directly to main)
```

File: docs/admin/outage-comms.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+#######################
+
+Intended audience: Anyone who is administering a Rubin Science Platform environment, or covers for an administrator.
+
+Putting up a banner
+===================
+
+The banners that appear on the RSP home page (:doc:`squareone </applications/squareone/index>`) are provided by the :doc:`semaphore </applications/semaphore/index>` service.
+
+The payload for the content of those banners is in the repo `rsp_broadcast <https://github.com/lsst-sqre/rsp_broadcast/>`_. You will need push access to that repository.
```
Comment (jonathansick) [inline docs/admin/outage-comms.rst:12]:
```suggestion
The content of those banners is sourced from the https://github.com/lsst-sqre/rsp_broadcast repository.
You will need push access to that repository.
```

"The payload of the content" is a weird turn of phrase, hence this suggestion.

The theme handles and displays full URLs to GitHub repos well.

File: docs/admin/outage-comms.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+1. Edit or create a message in https://github.com/lsst-sqre/rsp_broadcast/tree/main/broadcasts (if you are in a hurry and you have an issue affecting data.lsst.cloud consider editing 104_idfprod_outage.md on Github and committing directly to main)
+
+2. Edit your message. If you are starting from an existing message
+    a. Update the summary and body of the message (the body is what appears if the user expands
+    the banner)
+    b. Check the correct environment is listed in the message
+    c. Set ``enabled`` to true (if you're ready for the message to go live)
+
+3. Commit/push to main. The message will immediately appear on your RSP's landing page.
```
Comment (jonathansick) [inline docs/admin/outage-comms.rst:34]:
```suggestion
3. Commit and push to ``main``. The message will immediately appear on your RSP's landing page.
```

File: docs/admin/outage-comms.rst [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    We are aware of and are investigating an issue with DP0.2 searches in the portal
+
+The summary appears on the banner, the body appears if the user clicks the "details" button on the banner.
+See :doc:`environments </environments/index>` for the short labels for your environment.
+
+.. seealso::
+
+    More information on semaphore and the syntax of its payload repositories can be found at https://sqr-060.lsst.io.
+    Routine banners such as the Patch Thursday messages automatically go up and down using semaphore's cron-like syntax and you should not be editing them.
+    By default, banners are "scary red". For informational banners use ``category: info`` in the yaml header.
```
Comment (jonathansick) [inline docs/admin/outage-comms.rst:59]:
```suggestion
    By default, banners are "scary red". For informational banners use ``category: info`` in the YAML header.
```

**Pull request merged**